modules/fedimint-mint-server/src/lib.rs

```diff
@@ -734,7 +740,7 @@ mod test {
                     .unwrap()
                     .consensus
                     .peer_tbs_pks,
-                fee_consensus: FeeConsensus::default(),
+                fee_consensus: FeeConsensus::new_lnv2(1000).expect("Relative fee is within range"),
```

Comment:
Should this be `new_mint` instead of `new_lnv2`? Seems out of place to call other module logic from within the mint server.